src/app/pages/registration/register-user.jsx

```diff
@@ -78,17 +96,24 @@ function ChplRegisterUser({ hash }) {
               category: 'Authentication',
               event: 'Create New Account',
             });
-            setMessage('Your account has been created. Please check your email for your temporary password');
+            enqueueSnackbar('Your account has been created. Please check your email for your temporary password', {
+              variant: 'success',
+            });
             setState('login');
           },
           onError: (error) => {
-            if (error.response.data.errorMessages?.length > 0) {
-              setMessage(error.response.data.errorMessages[0]);
-            } else if (error.response.data.error) {
-              setMessage(error.response.data.error);
+            let errorMessage = ''; // Initialize the errorMessage variable
```

Comment:
```suggestion
            let errorMessage;
```

You don't need to initialize this, as long as you know it will be filled in before it's used